tools/docker/healthcheck/Dockerfile

```diff
@@ -1,16 +1,31 @@
 # Stage 1 - Build elixir release of ockam_healthcheck elixir app
-FROM ghcr.io/build-trust/ockam-builder@sha256:5ab42598e35509cad3ea9c1e1bd0ed135ed1340c6ae44b762b1c8bbec31d5c68 as elixir-app-release-build
+FROM hexpm/elixir:1.14.5-erlang-24.3.4.11-debian-bullseye-20230227-slim AS elixir-app-release-build
```

Comment:
Let's pin the image version `hexpm/elixir:1.14.5-erlang-24.3.4.11-debian-bullseye-20230227-slim@sha256:69a2436202ac4cd0078dff0758d11e0c55ed663fc8de8bdc9c7287a369912db4`